static/i18n/en/app.json

```diff
@@ -5,6 +5,7 @@
     "apply": "Apply",
     "confirm": "Confirm",
     "cancel": "Cancel",
+    "support": "Contact Support",
```

Comment:
to be confirmed with @dasilvarosa  but I think it's a "Contact us" wording today . at least it's the case in receive.confirmAddress.support so i think we probably want to share this (which would be out of the box if we could have a reused "ContactButton")

Reply:
Yep, either:
- Contact us --> support.ledgerwallet.com/hc/requests/new
- Get support --> support.ledgerwallet.com